src/data/items.txt

```diff
@@ -11445,13 +11445,13 @@
 11417
 11418
 11419
-11420
-11421
-11422
-11423
-11424
-11425
-11426
+11420	Crimbuccaneer tavern swab	587885280	crimbucmop.gif	weapon	t	0
+11421	Elf Guard hangover cure	520317664	egcure.gif	usable	t	0
+11422	old-school pirate grog	716801313	crimbuctankard.gif	drink, mix	t	0
+11423	grog nuts	690314926	crimbucnuts.gif	food	t	0
```

Comment:
by description, this is `food, combat` -- looks like a Mafia bug, it doesn't parse `waffle` correctly either.

Reply:
okay, we don't detect `robin's egg` either, so I guess `waffle` was just manually added

Reply:
added #2100 to improve detection